### DISCUSSION TITLE
Claude史诗级削弱

### DISCUSSION THREAD

**Discussion by Guin-Shalom** [viopsa233/AI-testers-Forum#67] [Claude]
今天试了下随身摄影机那个指令，应该是被玩坏了。
不管怎么reset都会果咩那塞
“I apologize, but I will not provide any responses that violate Anthropic's Acceptable Use Policy or could promote harm.”

**Comment by viopsa233** [comment]
不只这个,所有的模板差不多都被禁止了,开始关键字检测了

**Comment by viopsa233** [comment]
瑟瑟还是用newbing吧😋

**Reply by Guin-Shalom** [reply]
草 明明昨天还好用🥲

**Reply by firefoxmmx2** [reply]
不太懂，这个还可以瑟瑟？ 不就是做一些问答的对话吗

**Comment by TiansuoLi** [comment]
不是，好像是监视你的工作区了，会被强行介入替换回答。我的工作区用自己的咒语还能玩，可能是我玩的不够花？🤔

**Reply by Guin-Shalom** [reply]
这下得到官方认可了（）

**Reply by Guin-Shalom** [reply]
新建了个工作区，结果Claude不会说话🥲

**Reply by NostalgiaGit** [reply]
> 新建了个工作区，结果Claude不会说话🥲

现在新建的都这样，解决方法是在Slack app的工作区里点击Slack Connect，然后点创建新频道（Create a channel），这个功能是Pro的，免费用户只提供一段时间的试用期，创建完新Channel后点进那个频道，在里面@ Claude说句话，频道会问题是否邀请Claude，你点邀请，Claude就会响应并且弹出来一个用户协议，你点Agree同意，然后再回原来的App区就又能和那里的Claude正常对话了。

**Reply by Guin-Shalom** [reply]
先生真乃神人也。
不过我好像是黑号了，换了工作区还是在道歉😶‍🌫️

**Comment by ghost** [comment]
成功突破限制 https://github.com/viopsa233/AI-testers-Forum/discussions/42#discussioncomment-6192872
